cime_config/tests.py

```diff
@@ -282,6 +282,7 @@
             "SMS_D_Ld1.T62_oQU240wLI.GMPAS-IAF-PISMF.mpaso-impl_top_drag",
             "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-harmonic_mean_drag",
             "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-upwind_advection",
+            "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-freshwater_tracers",
```

Comment:
It might be worth considering:
```suggestion
            "SMS_D_Ld1.T62_oQU240wLI.GMPAS-IAF-PISMF.mpaso-freshwater_tracers",
```
to also test the ice-shelf-melt freshwater tracer.  What do you think?

Reply:
If we want full coverage of the tracers, we would need a DIB-*ISMF case because this case would use ice runoff instead of iceberg fluxes.

Reply:
Actually, we'd also need DSGR. Maybe add `PET_P480_Ld2.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF-DSGR.chrysalis_intel.mpaso-freshwater_tracers` to `e3sm_ocnice_extra_coverage`?

Reply:
I added `PET_P480_Ld2.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF-DSGR.chrysalis_intel.mpaso-freshwater_tracers` to `e3sm_ocnice_extra_coverage` and ran it successfully